package.json

```diff
@@ -30,8 +30,8 @@
     "build": "gulp build",
     "build-tsc": "tsc -p tsconfig.es5.json && tsc -p tsconfig.es6.json",
     "prepublish": "gulp default",
-    "test": "mocha --require ts-node/register --require source-map-support/register src/**/*.spec.ts",
-    "watch-tests": "mocha --watch --reporter min --require ts-node/register --require source-map-support/register src/**/*.spec.ts",
+    "test": "mocha --require ts-node/register --require source-map-support/register src/**.spec.ts src/**/*.spec.ts",
```

Comment:
you can remove the first pattern.
"src/**/*.spec.ts" already means "files ending in .spec.ts in src, with or without subdirectories"

Reply:
Yes I thought so too, but it only matched subdirectories of `src`, but not files in the `src` folder itself.
I googled it now, and I had the same problem as this guy: https://github.com/mochajs/mocha/issues/1828

Putting the argument in single quotes works now.